lib/engine/step/g_18_al/bye_sell_par_shares.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require_relative '../buy_sell_par_shares'
+
+module Engine
+  module Step
+    module G18AL
+      class BuySellParShares < BuySellParShares
+        def initialize
```

Comment:
you should never ovverride initialize, you can use def setup instead

Reply:
Done